api/fixtures/example.go

```diff
@@ -298,6 +300,30 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 		default:
 			panic(fmt.Sprintf("service publishing type %s is not supported", o.Kubevirt.ServicePublishingStrategy))
 		}
+		if o.Kubevirt.ExtClusterKubeConfig.KubeConfigContents != nil && o.Kubevirt.ExtClusterKubeConfig.ClusterName != "" {
+			extKubeConfigSecret := &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: o.Namespace,
+					Name:      o.Kubevirt.ExtClusterKubeConfig.ClusterName + "-kubeconfig",
+				},
+				Data: map[string][]byte{
+					"kubeconfig": []byte(base64.StdEncoding.EncodeToString(o.Kubevirt.ExtClusterKubeConfig.KubeConfigContents)),
+				},
+			}
```

Comment:
one more comment i missed in my previous review.

The secret we copy into the hosted control plane's namespace will automatically get cleaned up when the namespace is removed.

What cleans up this secret made in the hosted cluster's namespace though?

Reply:
I'll add an implementation for that secret deletion in the kubevirt destroy cmd.

Reply:
actually, there's already a mechanism that removes the secrets from the `clusters` namespace in the destroy command, as long as they are labeled with `hypershift.openshift.io/auto-created-for-infra=InfraID`:
https://github.com/openshift/hypershift/blob/main/cmd/cluster/core/destroy.go#L149

For that purpose, I populated the InfraID with a value at:
https://github.com/openshift/hypershift/pull/2017/files#diff-a8a4a70fd459a31d15b6500359d7e0e3cd47c5f7a48f551071756ddc395eaa20R92-R96